.whitesource

```diff
@@ -0,0 +1,8 @@
+##########################################################
```

Comment:
This whitesource config seems unrelated to Rubocop. I don't think that belongs in this PR.

Reply:
True. I can't remove it. If I could, it would be recreated.